### DISCUSSION TITLE
Peptidomics: quantifying peptides with different modifications

### DISCUSSION THREAD

**Discussion by kcasctivcr** [vdemichev/DiaNN#437] [Q&A] [answered]
Hi Vadim,

I recently performed non-specific peptidomic quantification analysis with DIA-NN, however I ran into some troubles

This is what I have done so far:
Mascot search with DDA data to get the peptide sequence list
Generate a fasta file based on the peptide sequences for the construction of spectral library with DIA-NN
Using this spectral library to quantify peptides with DIA-NN

I tested this procedure with a single DIA data file, and I found that peptides with same stripped sequence but different modifications all showed same expression level
![image](https://user-images.githubusercontent.com/65659359/179448012-0748f0a3-81ab-4f30-9be7-a663e5c8e752.png)
I suppose that something has gone wrong?
Following is the log files of library generation and quantification analysis
[library.log.txt](https://github.com/vdemichev/DiaNN/files/9129525/library.log.txt)
[output.log.txt](https://github.com/vdemichev/DiaNN/files/9129526/output.log.txt)

Many thanks,
Hung

**Comment by vdemichev** [comment, marked as answer]
Hi Hung,

PG.Normalised & PG.Quantity are protein quantities, they are expected to be the same for peptides with the same sequence, so all good here.

Best,
Vadim

**Reply by kcasctivcr** [reply]
Got it, but is there a way to distinguish the quantities between peptides with same sequence but different modifications?
Like I mentioned, I was trying to perform peptidomic quantitative analysis. Quantities of peptides in different modified state may include crucial information, could this possibly be done by DIA-NN?

![image](https://user-images.githubusercontent.com/65659359/179903730-d8811b3c-e783-4836-8ebb-cfb4da3c685f.png)

Precursor quantity & precursor normalised showed the same values as well.

Best,
Hung

**Reply by kcasctivcr** [reply]
Hi Vadim,

Is there any suggestions? About how to quantify differently modified peptides but with same stripped sequence (the screenshot above).

Best,
Hung

**Reply by vdemichev** [reply]
Hi Hung,

Need to use --monitor-mod and filter by the PTM-related scores.

Best,
Vadim

**Reply by kcasctivcr** [reply]
Hi Vadim,

(1) What's the suggestion of filtering? I suppose is to filter by PTM.Q.Value?
(2) And what is PTM.Site.Confidence?
(3) I notice many peptides without modifications have high PTM.Site.Confidence values, what does this suppose to mean?
(4) I thought specifying -monitor-mod would only calculate PTM -related scores additionally, but I found that upon specifying this also cause difference in quantity-related values, why?

Many thanks,
Hung

**Reply by kcasctivcr** [reply]
Hi Vadim,

Looking forward for some answers of my previous questions :D

Many thanks,
Hung

**Reply by vdemichev** [reply]
(1) Or a single/combination of any of the PTM scores; there's no universal recipe, choice depends on the experiment
(2) Confidence that occupancy of all PTM sites on the peptide is reported correctly
(3) It's a score meant to be used as a filter: no mods => it's not relevant
(4) It affects search decisions

**Reply by kcasctivcr** [reply]
Hi Vadim,

Thanks for your patient explanations, really appreciate it!

Best,
Hung
